demo/controllers/dataTableController.ts

```diff
@@ -30,6 +31,8 @@ export default class DataTableController {
       let start = (this.settings.current - 1) * this.settings.perpage;
       this.onLoadNext(start, this.settings.perpage);
       this.onSort(2, true);
+      this.settings.selectAllTitle = 'Select All';
```

Comment:
These are in demo app, you have to send `selectAllTitle` and `sortedByTitle` from outside app. This way we can use i18 in MiQ.

But we should definitely add i18 to this.